src/components/inputs/Upload.tsx

```diff
@@ -84,7 +84,10 @@ export const GenericImageUpload = <FormValues extends FieldValues>({
   const [zoom, setZoom] = useState(1);
   const [imageFile, setImageFile] = useState<File | undefined>(undefined);
   const [croppedAreaPixels, setCroppedAreaPixels] = useState(null);
-
+  const ratioFloat = ratio
+    ?.split(':')
+    .map((value) => parseInt(value))
```

Comment:
Denne kan kortes ned :)
```suggestion
    .map(Number)
```